plugins/wazuh-core/public/plugin.ts

```diff
@@ -19,12 +20,21 @@ export class WazuhCorePlugin
   services: { [key: string]: any } = {};
   public async setup(core: CoreSetup): Promise<WazuhCorePluginSetup> {
     const noop = () => {};
-    const logger = {
+    // Debug logger
+    const consoleLogger = {
+      info: console.log,
+      error: console.error,
+      debug: console.debug,
+      warn: console.warn,
+    };
+    // No operation logger
+    const noopLogger = {
```

Comment:
You can create a new file for each `Logger` inside the new folder called `logger`.

Reply:
We will need to discuss about the frontend logger. For now, I will leave the code in that state.